### DISCUSSION TITLE
Is there a url to show diff with two specified file？

### DISCUSSION THREAD

**Discussion by m986883511** [coder/code-server#6484] [General]
I am write note, I want to add a code-server url, to show two file diff.
Is there a way? 

another way? I can install a plugin, may be named "diff & merge", how to use url to call the plugin?

I want to another people click the url, then open brower to watch the diff between two files.

**Comment by code-asher** [comment]
I am not sure.  There might be a way to do it through the payload but someone would need to experiment.  https://github.com/coder/code-server/issues/1964#issuecomment-1546098145

**Reply by m986883511** [reply]
yes, maybe playlaod can do it, do you known playload support command? expect openFile, gotoLineMode...

**Reply by code-asher** [reply]
I have no idea, unfortunately they use an untyped `object` so the only way to figure it out is to read through the code and see what it does with the payload.
